tests/gis_tests/geoapp/test_functions.py

```diff
@@ -223,12 +223,9 @@ def test_intersection(self):
         geom = Point(5, 23, srid=4326)
         qs = Country.objects.annotate(inter=functions.Intersection('mpoly', geom))
         for c in qs:
-            if spatialite or mysql:
-                # When the intersection is empty, Spatialite and MySQL return None
+            if spatialite or mysql or oracle:
```

Comment:
This could be a separate commit or ticket, but could a database feature flag replace the vendor checks?